### PR TITLE
feature/cbb

### PR DESCRIPTION
**Pull request by johannwagner**

### CODE REVIEW

File: cosmo/netboxclient.py [hunk 1]
```diff
@@ -375,7 +375,7 @@ def get_data(self, device_config):
         r = self.query(query)
         graphql_data = r['data']
 
-        static_routes = self.query_rest("/api/plugins/routing/staticroutes/", {"device": device_list})
+        static_routes = self.query_rest("api/plugins/routing/staticroutes/", {"device": device_list})
```
Comment (loulecrivain) [inline cosmo/netboxclient.py:378]: why do we have this change?

File: cosmo/serializer.py [hunk 2]
```diff
@@ -66,9 +66,16 @@ def render_addresses(self):
         retVal.update(secondaryIPs)
         return retVal
 
+class RouterSerializerConfig:
+    def __init__(self, config = {}):
+        self.config = config
+
+    @property
```
Comment (loulecrivain) [inline cosmo/serializer.py:73]: For now it's good but maybe we should implement something more generic for the future ? like __getattr__ or something. Not sure if we really need to do this, it's up to you.

**Review by loulecrivain — COMMENTED** [review]
small changes are needed (?) (not sure), but overall looks good :+1:

File: cosmo/netboxclient.py (hunk 1 above)
Comment (johannwagner) [reply cosmo/netboxclient.py:378]: `query_rest` already appends a `/` to the base URL.
We debugged stuff and wondered, if the double slash was a problem - normally its not - and it was not the problem.

File: cosmo/serializer.py (hunk 2 above)
Comment (johannwagner) [reply cosmo/serializer.py:73]: I need a default somehow.

File: cosmo/netboxclient.py (hunk 1 above)
Comment (loulecrivain) [reply cosmo/netboxclient.py:378]: yea no its not. I think we should use more of urllib in the future (maybe this is something that can be adressed for future refactor?)

**Pull request merged**